lib/wrapbox/runner/ecs.rb

```diff
@@ -506,18 +508,26 @@ def build_run_task_options(task_definition_arn, command, environments, ec2_insta
         if ec2_instance_id
           additional_placement_constraints << { type: "memberOf", expression: "ec2InstanceId == #{ec2_instance_id}" }
         end
-        {
+        options = {
           cluster: @cluster,
           task_definition: task_definition_arn,
           overrides: overrides,
           placement_strategy: placement_strategy,
           placement_constraints: placement_constraints + additional_placement_constraints,
-          launch_type: @launch_type,
           network_configuration: network_configuration,
           started_by: "wrapbox-#{Wrapbox::VERSION}",
           enable_ecs_managed_tags: enable_ecs_managed_tags,
           propagate_tags: propagate_tags,
         }
+        if @capacity_provider_strategy.empty?
+          options[:launch_type] = @launch_type if @launch_type
+        else
+          if @launch_type
+            @logger.warn("#{log_prefix}Ignore --launch_type option when specified capacity_provider_strategy in the configuration file")
```

Comment:
Is this correct? It seems to be expected to raise an error as I check its value of "What's happened" column in the table of the description.

Reply:
Fixed in 1f41ca3

Reply:
Ah, I meant I didn't know which was correct, raising an error, as described in the description, or displaying a warning.
By the way, `@launch_type` is always equal to `options[:launch_type]` because `options[:launch_type` is assigned to `@launch_type`. In other words, if `@launch_type` is truthy, `options[:launch_type].nil?` is always false.

Reply:
I misunderstood.
An error will be raised from AWS SDK if we set values to both `launch_type` and `capacity_provider_strategy` in config.yml.
A warning will be displayed if we set values to both `--launch_type` via command line and `capacity_provider_strategy` via config.yml.

Reply:
I checked you had modified the description and the warning message 👍